app/models/user.rb

```diff
@@ -6,6 +6,18 @@ class User < ApplicationRecord
     has_many :post_likes
     has_many :work_experiences
 
+    has_many :blogs
+    has_many :articles
+    has_many :article_comments
+    has_many :article_likes
+
+    # FOllowers relationship
+    has_many :followers, foreign_key: :user_id, class_name: 'Follower'
+    has_many :followed_users, through: :followers, source: :follower_user
+
+    has_many :followings, foreign_key: :user_id, class_name: 'Following'
+    has_many :following_users, through: :followings, source: :following_user
+
```

Comment:
#### Add followers_count column to the users table and give it a default value of 0 so as to prevent errors while updating